clearml-server-cloud-ready/README.md

```diff
@@ -0,0 +1,163 @@
+# ClearML Server for Kubernetes Clusters Using Helm
+
+##  Auto-Magical Experiment Manager & Version Control for AI
+
+[![GitHub license](https://img.shields.io/badge/license-SSPL-green.svg)](https://img.shields.io/badge/license-SSPL-green.svg)
+[![GitHub version](https://img.shields.io/github/release-pre/allegroai/clearml-server.svg)](https://img.shields.io/github/release-pre/allegroai/clearml-server.svg)
+[![PyPI status](https://img.shields.io/badge/status-beta-yellow.svg)](https://img.shields.io/badge/status-beta-yellow.svg)
+
+## Introduction
+
+The **clearml-server** is the backend service infrastructure for [ClearML](https://github.com/allegroai/clearml).
+It allows multiple users to collaborate and manage their experiments.
+By default, *ClearML is set up to work with the ClearML Demo Server, which is open to anyone and resets periodically. 
+In order to host your own server, you will need to install **clearml-server** and point ClearML to it.
+
+**clearml-server** contains the following components:
+
+* The ClearML Web-App, a single-page UI for experiment management and browsing
+* RESTful API for:
+    * Documenting and logging experiment information, statistics and results
+    * Querying experiments history, logs and results
+* Locally-hosted file server for storing images and models making them easily accessible using the Web-App
+
+Use this repository to add **clearml-server** to your Helm and then deploy **clearml-server** on Kubernetes clusters using Helm.
+ 
+## Prerequisites
+
+* a Kubernetes cluster
+* `kubectl` is installed and configured (see [Install and Set Up kubectl](https://kubernetes.io/docs/tasks/tools/install-kubectl/) in the Kubernetes documentation)
+* `helm` installed (see [Installing Helm](https://helm.sh/docs/using_helm/#installing-helm) in the Helm documentation)
+* one node labeled `app: clearml`
```

Comment:
Is this still true for the cloud-ready chart?

Reply:
Also, perhaps add some notes on the need for PVs for MongoDB/ElasticSearch/Redis